charmcraft/utils.py

```diff
@@ -33,6 +39,20 @@ def make_executable(fh):
     os.fchmod(fileno, mode)
 
 
+def load_yaml(fpath):
+    """Return the content of a YAML file."""
+    if not fpath.is_file():
+        logger.debug("Couldn't find config file %s", fpath)
+        return
+    try:
+        with fpath.open('rb') as fh:
+            content = yaml.safe_load(fh)
+    except (yaml.error.YAMLError, OSError) as err:
+        logger.error("Failed to read/parse config file %s (got %r)", fpath, err)
```

Comment:
Perhaps simply displaying the original error message after a colon would be easier to read ("Failed to do something: this bad thing happened").

Reply:
nice, improved